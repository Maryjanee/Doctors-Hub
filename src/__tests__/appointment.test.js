import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Appointment from '../components/Appointments';
import store from '../Store';

it('Appointments', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Appointment />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
