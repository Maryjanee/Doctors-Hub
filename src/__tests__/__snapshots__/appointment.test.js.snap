// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appointments 1`] = `
<div
  className="appointments"
>
  <h2
    className="text-center"
  >
    All Appointments
  </h2>
  <h4
    className="text-center"
  >
    Hello
     
    You have no appointments
  </h4>
  <div
    className="appointment-container"
  />
</div>
`;
